columnflow/tasks/framework/mixins.py

```diff
@@ -254,9 +254,9 @@ def store_parts(self):
 class SelectorStepsMixin(SelectorMixin):
 
     selector_steps = law.CSVParameter(
-        default=(),
+        default=("default",),
         description="a subset of steps of the selector to apply; uses all steps when empty; "
-        "empty default",
+        "default: (default,)",
```

Comment:
does it really make sense to add a default here? The empty default also uses the "main" selection when calling ReduceEvents

Reply:
Also, I think we wantd to use the `selector-steps` parameter after the reduction to perform an additional last-minute selection before e.g. plotting, right?

Reply:
I forgot about that, you're right! Reverted this one.